tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -201,9 +201,7 @@ export class WebGPUBackend extends KernelBackend {
    * @oaram force Optional, remove the data regardless of refCount
    */
   override disposeData(dataId: DataId, force = false): boolean {
-    if (this.tensorDataPendingDisposal.indexOf(dataId) >= 0) {
```

Comment:
I prefer that the code is not removed here. If removing this code, the array of `tensorDataPendingDisposal` could store multiple same copies `dataId`. I know the below code could handle the correctness that user calls dispose api sometimes. But why not directly return here.